extensions/commands/art/cmd_remote.py

```diff
@@ -0,0 +1,184 @@
+import base64
+import getpass
+import json
+import os.path
+import requests
+
+from conan.api.conan_api import ConanAPI
+from conan.api.output import ConanOutput
+from conan.cli.command import conan_command, conan_subcommand
+from conan.errors import ConanException
+
+REMOTES_FILE = "art-remote.conf"
```

Comment:
That's because we decided to obfuscate the place where we save the token and the data a bit, using a different extension or no extension and storing the json in base64

Reply:
Oh, I see. If that is the case, I'd directly remove the .conf extension, and make it private with ``.art-servers``

Reply:
renamed!